generic_tracers/generic_tracer_utils.F90

```diff
@@ -3395,8 +3504,13 @@ subroutine g_tracer_vertdiff_G(g_tracer, h_old, ea, eb, dt, kg_m2_to_H, m_to_H,
       do j = g_tracer_com%jsc, g_tracer_com%jec
          do i = g_tracer_com%isc, g_tracer_com%iec
             do k = 1, g_tracer_com%nk
-               g_tracer%vdiffuse_impl(i,j,k) = g_tracer_com%grid_tmask(i,j,k) *   &
-                    (g_tracer%field(i,j,k,tau) - g_tracer%vdiffuse_impl(i,j,k)) / dt
+               !liao: let h_old into vdiffuse and this term is content tendency
```

Comment:
Do we want to remove liao 's name here too?

Reply:
We could. I left it in thinking it was the one case where there was an actual comment included with the name